### PR TITLE
Added logical definitions for hormone secretion and endocrine hormone…

### PR DESCRIPTION
**Pull request by raymond91125**
… secretion. #17084

### CODE REVIEW

File: src/ontology/go-edit.obo [hunk 1]
```diff
@@ -278773,6 +278773,8 @@ namespace: biological_process
 def: "The regulated release of hormones, substances with a specific regulatory effect on a particular organ or group of cells." [ISBN:0198506732]
 is_a: GO:0009914 ! hormone transport
 is_a: GO:0023061 ! signal release
+intersection_of: GO:0046903 ! secretion
+intersection_of: has_primary_input CHEBI:24621 ! hormone
```
Comment (dosumis) [inline src/ontology/go-edit.obo:278777]: @cmungall - should this be using the GOCHE term for molecule with hormone role?

Comment (balhoff) [reply src/ontology/go-edit.obo:278777]: I think so. Sadly the checks aren't failing.

Comment (raymond91125) [reply src/ontology/go-edit.obo:278777]: When and how should we use GOCHE vs CHEBI?

Comment (balhoff) [reply src/ontology/go-edit.obo:278777]: Any time you need to use a CHEBI role in a place where a material entity is needed. In CHEBI, 'hormone' is classified under role.

Comment (raymond91125) [reply src/ontology/go-edit.obo:278777]: I see. Thanks.

**Pull request closed without merging**